main/2024/CG-06.md

```diff
@@ -74,7 +74,7 @@ scheduled times if previous sessions end early.
 - 2:00pm: JS string builtins (Ryan Hunt)
     - Vote to advance to phase 3
 - 2:45pm: Break
-- 3:00pm: Memory control (Ben Visness and Deepti Gandluri)
+- 3:00pm: Memory control (Deepti Gandluri)
```

Comment:
I think these agenda diffs are reverts of the agenda changes made after this file landed in the repo but before the meeting.

Reply:
Thanks, fixed. I left off the link to Luke's slides because it appears again alongside the notes for that session.